### ISSUE TITLE
MODEL WEIGHTS

### ISSUE THREAD

**Issue by JinChen0056** [Sherrylone/PQDiff#2]
MUST WE TRAIN OWN MODEL? WILL YOU RELEASE YOUR MODEL WEIGHTS?THANKS~

**Comment by Sherrylone** [comment]
Yes, perhaps we will release the pretrained model weights in the next month.

**Comment by dawei03896** [comment]
+1

**Comment by dawei03896** [comment]
When to release the pretrained model weights? It's April now！

**Comment by inFaaa** [comment]
Thanks for your interest. The old model weights are on company equipment and we are reproducing the new model weights. We will release the model weight before May 2024.

**Comment by chl916185** [comment]
Building Facades，the pretrained model weights?
@inFaaa

**Comment by ustczhouyu** [comment]
> Thanks for your interest. The old model weights are on company equipment and we are reproducing the new model weights. We will release the model weight before May 2024.

Why are the Sampling Results by the Provided Pre-trained Model and My model trained by myself Colorful and Chaotic? My Sampling Command Is CUDA_VISIBLE_DEVICES=0 python evaluate.py --target_expansion 0.25 0.25 0.25 0.25 --eval_dir. /eval_dir/scenery/2x/ --size 128 --config flickr192_large. Did anyone get the correct sampling results?
